synfig-studio/src/gui/app.cpp

```diff
@@ -1349,19 +1370,22 @@ DEFINE_ACTION("keyframe-properties", _("Properties"));
 	ACCEL2(Gtk::AccelKey(')',Gdk::CONTROL_MASK,					"<Actions>/canvasview/increase-low-res-pixel-size"			));
 	ACCEL2(Gtk::AccelKey('(',Gdk::MOD1_MASK|Gdk::CONTROL_MASK,			"<Actions>/action_group_layer_action_manager/amount-dec"		));
 	ACCEL2(Gtk::AccelKey(')',Gdk::MOD1_MASK|Gdk::CONTROL_MASK,			"<Actions>/action_group_layer_action_manager/amount-inc"		));
-	ACCEL2(Gtk::AccelKey(']',Gdk::CONTROL_MASK,					"<Actions>/canvasview/jump-next-keyframe"				));
-	ACCEL2(Gtk::AccelKey('[',Gdk::CONTROL_MASK,					"<Actions>/canvasview/jump-prev-keyframe"				));
 	ACCEL2(Gtk::AccelKey('=',Gdk::CONTROL_MASK,					"<Actions>/canvasview/canvas-zoom-in"					));
 	ACCEL2(Gtk::AccelKey('-',Gdk::CONTROL_MASK,					"<Actions>/canvasview/canvas-zoom-out"					));
 	ACCEL2(Gtk::AccelKey('+',Gdk::CONTROL_MASK,					"<Actions>/canvasview/time-zoom-in"					));
 	ACCEL2(Gtk::AccelKey('_',Gdk::CONTROL_MASK,					"<Actions>/canvasview/time-zoom-out"					));
-	ACCEL2(Gtk::AccelKey('.',Gdk::CONTROL_MASK,					"<Actions>/canvasview/seek-next-frame"					));
-	ACCEL2(Gtk::AccelKey(',',Gdk::CONTROL_MASK,					"<Actions>/canvasview/seek-prev-frame"					));
-	ACCEL2(Gtk::AccelKey('>',Gdk::CONTROL_MASK,					"<Actions>/canvasview/seek-next-second"					));
-	ACCEL2(Gtk::AccelKey('<',Gdk::CONTROL_MASK,					"<Actions>/canvasview/seek-prev-second"					));
+	ACCEL("bracketleft",             "<Actions>/canvasview/jump-prev-keyframe");
+	ACCEL("bracketright",            "<Actions>/canvasview/jump-next-keyframe");
+	ACCEL("comma",                   "<Actions>/canvasview/seek-prev-frame");
+	ACCEL("period",                  "<Actions>/canvasview/seek-next-frame");
+	ACCEL("<Shift>less",             "<Actions>/canvasview/seek-prev-second");
+	ACCEL("<Shift>greater",          "<Actions>/canvasview/seek-next-second");
+	ACCEL("<Control><Shift>less",    "<Actions>/canvasview/seek-begin");
+	ACCEL("<Control><Shift>greater", "<Actions>/canvasview/seek-end");
 	ACCEL("<Mod1>o",								"<Actions>/canvasview/toggle-onion-skin"				);
 	ACCEL("<Control><Shift>z",							"<Actions>/canvasview/canvas-zoom-fit"					);
-	ACCEL("<Control>p",								"<Actions>/canvasview/play"						);
+	ACCEL("space",                   "<Actions>/canvasview/play");
+	ACCEL("space",                   "<Actions>/canvasview/pause");
```

Comment:
I did not try your PR yet, but it seems to be my last comment before it :D : 
Does it work in current code? Same shortcuts for two 'different' actions?

Reply:
Yes, it works.

My guess is when we hit space synfig tries to determine if the animation is playing to pause it but it's not so `play` action is called. The second time we hit space synfig tries to determine the same and finds out animation is playing right now, `pause` action gets called.

I wanted to use the same action name too like this `play-pause` but that didn't work.